### PR TITLE
Feat: Support for Pinning to Skynet using single account

### PR DESCRIPTION
**Pull request by jay-dee7**
Signed-off-by: jay-dee7 <jasdeepsingh.uppal@gmail.com>

### CODE REVIEW

File: skynet/skynet.go [hunk 1]
```diff
@@ -39,19 +49,29 @@ func (c *Client) Upload(namespace, digest string, content []byte, headers ...sky
...
 	data[digest] = buf
 
-	return c.skynet.Upload(data, opts, headers...)
+	skylink, err := c.skynet.Upload(data, opts)
+	if err != nil {
+		return "", err
+	}
+
+	if pin {
+		time.Sleep(time.Second * 10)
```
Comment (guacamole) [inline skynet/skynet.go:58]: @jay-dee7 , is the sleep really needed here?

File: skynet/skynet.go [hunk 2]
```diff
@@ -101,6 +128,7 @@ func (c *Client) Metadata(skylink string) (uint64, bool) {
 	if err != nil {
 		return 0, false
 	}
+	color.Red(skl, contentLength)
```
Comment (guacamole) [inline skynet/skynet.go:131]: can we remove this color.red?

Comment (jay-dee7) [reply skynet/skynet.go]: done

File: skynet/skynet.go (hunk 1 above)
Comment (jay-dee7) [reply skynet/skynet.go]: i tried removing it and it works fine now. marking it as resolved

**Review by guacamole — APPROVED** [review]
LGTM 🥇

**Pull request merged**